test/integration_test_helper.rb

```diff
@@ -124,6 +124,13 @@ def set_request_user(user)
     user = users(user) unless user.is_a?(User)
     create_cookie('test_user', user.login)
   end
+
+  def with_controller_caching(*controller_klasses)
+    controller_klasses.each { |c| c.perform_caching = true }
+    yield
+  ensure
+    controller_klasses.each { |c| c.perform_caching = false }
+  end
 end
```

Comment:
Nice, I just noticed we use `:file_store` for test environment:

    config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")

Do you think let's say `ActiveSupport::Cache::MemoryStore` might be more effective? We don't have many of those I assume, just the three here for now. But for the future.

Other than that, RTM.